public/pages/Findings/components/FindingDetailsFlyout.tsx

```diff
@@ -62,6 +69,28 @@ export default class FindingDetailsFlyout extends Component<
     );
   };
 
+  showRuleDetails = (fullRule, ruleId: string) => {
+    this.setState({
+      ...this.state,
+      ruleViewerFlyoutShown: true,
+      ruleViewerFlyoutData: {
+        ruleId: ruleId,
+        title: fullRule.title,
+        level: fullRule.level,
+        category: fullRule.category,
+        description: fullRule.description,
+        source: fullRule.source,
+        ruleInfo: {
+          _source: fullRule,
+        } as any,
```

Comment:
Why do we need to use `as any` here?

Reply:
I've changed it to `as RuleItemInfoBase` which is more appropriate.

TS assertion is needed because we don't have all the data to construct whole object. Other properties are anyway not in use in the Rule flyout.

Let me know if you have some better idea.